ObjectDetection:
  maxIterations: 1000
  minClusterPoints: 2000

  # Points need to be 5 cm together for a plane
  distanceThresholdPlane: 0.05
  # Value for simluation. Strange effects happens
  #distanceThresholdPlane: 0.095

  # Points need to be 15 cm together for a plane
  distanceThresholdCluster: 0.15

  # Continue until less then point * remainingPoints remain
  remainingPoints: 0.1

  # minimal size of objects
  minXDistance: 0.25
  minYDistance: 0.25

  # Maximal angle deviation from (0,0,1)
  groundAngleDelta: 5.0
  boxAngleDelta: 7.5
  wedgeAngleDelta: 12.5

  # upper (ground) and lower (box, wedge) bound for max.z of the plan 
  groundDistDelta: 0.1
  boxDistDelta: 0.2
  wedgeDistDelta: 0.2

  # World frame for transformation
  world_frame: base_footprint
  #world_frame: map
